src/golang/cmd/server/queries/sqlite.go

```diff
@@ -1,9 +1,66 @@
 package queries
 
+import (
+	"context"
+	"fmt"
+
+	"github.com/aqueducthq/aqueduct/lib/collections/operator"
+	"github.com/aqueducthq/aqueduct/lib/database"
+	"github.com/aqueducthq/aqueduct/lib/database/stmt_preparers"
+	"github.com/dropbox/godropbox/errors"
+	"github.com/google/uuid"
+)
+
 type sqliteReaderImpl struct {
 	standardReaderImpl
 }
 
 func newSqliteReader() Reader {
 	return &sqliteReaderImpl{standardReaderImpl{}}
 }
+
+func (r *sqliteReaderImpl) GetLoadOperatorSpecByOrganization(
+	ctx context.Context,
+	organizationId string,
+	db database.Database,
+) ([]LoadOperatorSpecResponse, error) {
+	query := fmt.Sprintf(
+		`SELECT DISTINCT workflow_dag_edge.from_id AS artifact_id, artifact.name AS artifact_name, operator.id AS load_operator_id, workflow.name AS workflow_name, workflow.id AS workflow_id, operator.spec 
+		 FROM app_user, workflow, workflow_dag, workflow_dag_edge, operator, artifact
+		 WHERE app_user.id = workflow.user_id AND workflow.id = workflow_dag.workflow_id AND 
+		 workflow_dag.id = workflow_dag_edge.workflow_dag_id AND workflow_dag_edge.to_id = operator.id AND 
+		 artifact.id = workflow_dag_edge.from_id AND
+		 json_extract(operator.spec, '$.type') = '%s' AND app_user.organization_id = $1;`,
```

Comment:
Nit: Instead of mixing the `%s` and `$1` syntax, can we use just the placeholders `$1` in both places here?

Reply:
We could, but I somehow found using `%s` for constant owned by us, and `$1` for user parameters is a pattern through out the db codes: https://github.com/aqueducthq/aqueduct/blob/main/src/golang/lib/collections/artifact/standard.go#L86 . I find this is a quite clean cut to distinguish the usage between the two. Esp. that some `%s` is necessary, I will add more details in the next comment down

Reply:
An alternative is to restrict all `%s` use to indexing and force all other use cases to use parameters.

Reply:
Got it, I think we can create a task for this for now and come back to it.

Reply:
https://linear.app/aqueducthq/issue/ENG-1367/make-better-use-of-query-parametrization